src/app/faucet/layout.tsx

```diff
@@ -0,0 +1,23 @@
+"use client";
```

Comment:
This layout is only being used for one page I am assuming?
If so, lets move it to the page.tsx itself. We dont need it for any other page